lib/v2/curius/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'curius.app': {
+        _name: 'Curius',
+        '.': [
+            {
+                title: 'Curius',
```

Comment:
```suggestion
                title: '用户',
```
`title` here should match H3 heading